src/databricks/labs/remorph/reconcile/compare.py

```diff
@@ -168,12 +176,17 @@ def _get_mismatch_df(source: DataFrame, target: DataFrame, key_columns: list[str
     filter_columns = " and ".join([column + "_match" for column in column_list])
     filter_expr = ~expr(filter_columns)
 
+    source.printSchema()
+    target.printSchema()
+    logger.info(f"KEY COLUMNS: {key_columns}")
+    logger.info(f"FILTER COLUMNS: {filter_expr}")
+    logger.info(f"SELECT COLUMNS: {select_expr}")
+
     mismatch_df = (
-        source.alias('base')
-        .join(other=target.alias('compare'), on=key_columns, how="inner")
-        .select(*select_expr)
-        .filter(filter_expr)
+        source.alias('base').join(other=target.alias('compare'), on=key_columns, how="inner").select(*select_expr)
     )
+    mismatch_df.show()
```

Comment:
this should be behind a debug flag
```suggestion
    mismatch_df.show()
```

Reply:
should I remove the statement?

Reply:
removed the debug message.